FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -7,18 +7,28 @@ import Foundation
 public final class RemoteFeedLoader: FeedLoader {
 	private let url: URL
 	private let client: HTTPClient
+	private let mapper: FeedImageMapper
 
 	public enum Error: Swift.Error {
 		case connectivity
 		case invalidData
 	}
 
-	public init(url: URL, client: HTTPClient) {
+	public init(url: URL, client: HTTPClient, mapper: FeedImageMapper = RemoteFeedImageMapper.default) {
 		self.url = url
 		self.client = client
+		self.mapper = mapper
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let self = self else { return }
```

Comment:
Following from my next comment, there would be no reason to unwrap `self`, so it can be a simple nullability check
```suggestion
			guard self != nil else { return }
```